api/src/main/java/au/org/aodn/nrmn/restapi/model/db/StagedJobLog.java

```diff
@@ -37,7 +25,7 @@
     @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "staged_job_log_id_seq")
     private Long id;
 
-    @ManyToOne
+    @ManyToOne(cascade = CascadeType.ALL)
```

Comment:
Did you need this? I don't think its required. Normally cascade is used on the parent only.